app/admin/nat/edit-result.php

```diff
@@ -56,7 +56,7 @@
     "dst_port"    => $_POST['dst_port'],
     "device"      => $_POST['device'],
     "description" => $_POST['description'],
-    "policy"      => "No",
+    "policy"      => _("No"),
```

Comment:
Hi, 

The $values[] array is sent to the database via object_modify(), $values["policy"] corresponds to a "Yes"/"No" SET in the SCHEMA (probably should be an ENUM). If translated the query will fail.

This is a common pattern in the code for arrays called $values.

https://github.com/phpipam/phpipam/blob/c913895f37c3fb2e924d61550844309d5ca457dd/app/admin/nat/edit-result.php#L82-L88

``` sql
CREATE TABLE `nat` (
  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `name` varchar(64) DEFAULT NULL,
  `type` set('source','static','destination') DEFAULT 'source',
  `src` text DEFAULT NULL,
  `dst` text DEFAULT NULL,
  `src_port` int(5) DEFAULT NULL,
  `dst_port` int(5) DEFAULT NULL,
  `device` int(11) unsigned DEFAULT NULL,
  `description` text DEFAULT NULL,
  `policy` set('Yes','No') NOT NULL DEFAULT 'No',
  `policy_dst` varchar(255) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Reply:
Thanks for explanation and your patience!
I reverted this part and added a small comment about non l10n strings here.

I'm not sure if in https://github.com/phpipam/phpipam/pull/3141/files#diff-e1ac431de9c2490bc62d15d5d1470431 is a similar situation.

Reply:
Same situation in app/admin/users/edit-result.php
```php
				"mailNotify"     =>$_POST['mailNotify'],
				"mailChangelog"  =>$_POST['mailChangelog'],
				"theme"          =>$_POST['theme']=="default" ? "" : $_POST['theme'],
-				"disabled"       =>$_POST['disabled']=="Yes" ? "Yes" : "No"
+				"disabled"       =>$_POST['disabled']==_("Yes") ? _("Yes") : _("No")
				);
```
The text displayed in the browser to the user is translated but the $_POST contains the `<select>` value=XXXX. This is "Yes" or "No" and matches the db schema ENUM.

$_POST['disabled'] = 

![image](https://user-images.githubusercontent.com/18753294/93267844-2ecb6400-f7a4-11ea-9fb5-166395d43b2c.png)

Reply:
Hi

If case you haven't noticed. The phpIPAM php code looks much better with tabs=4 spaces in your editor/git settings. It usually defaults to 8 spaces.

Don't worry about fixing formatting and whitespace too much for I10n. It makes it harder to work out what has changed when reviewing and changing one line at a time will take forever.

There are tools available to auto-format everything in bulk and add to a git blame ignore list (possible future todo)

Reply:
> Same situation in app/admin/users/edit-result.php

Yes, that's what I already was thinking and tried to show by the link to the diff. Will revert this to later too.

I'm aware about the problems that indentation only commits bringing in and the later usage of git blame / bisect so I try to do such things more conservative than progressive.
I'm o.k. if you say "don't change indentations in case of doing l10n work". The most projects I've worked on don't want any indentation only changes and allowing them only while working on the source code and that's fine. Basically I understand what the PHP code is doing around the strings, but sometimes it's hard to follow the logic as some parts are quite "interesting" indented. A lot of mixed up usage of tabs and spaces. :)
BTW: The editor I use (mostly gedit) is using exact this combination you suggested for displaying the source.

Reply:
> >Same situation in app/admin/users/edit-result.php
>
>Yes, that's what I already was thinking and tried to show by the link to the diff. Will revert this to later too.

Modified and force pushed again.